docs/release.md

```diff
@@ -39,28 +39,45 @@ These artifacts are provided in their respective channels and their natural way
 of distribution.
 
 # Cadence and Timeline
-Both a new release branch and initial release candidate are cut on the **first
-business day of each month.**
-
-If no blocker issues are discovered the release candidate is **promoted to a full
-release after 5 business days.**
+New release branches are cut 3 times a year, so a release cycle will occupy about 15 weeks.
+This release cycle mirrors the Kubernetes' and allows KubeVirt to release after the community.
+This looks like:
+| Week Number in Year | K8s Release Number | K8s Release Week | KubeVirt Release Week |                     Note                    |
+|:-------------------:|:------------------:|:----------------:|-----------------------|:-------------------------------------------:|
+| 1                   | 1.24               | 1 (January 03)   | 1 (~January 03)       |                                             |
+| 15                  | 1.24               | 15 (April 12)    | 15 (~April 12)        |                                             |
+| 17                  | 1.25               | 1 (April 26)     | 1 (~April 26)         | KubeCon + CloudNativeCon EU likely to occur |
+| 32                  | 1.25               | 15 (August 09)   | 15 (~August 09)       |                                             |
+| 34                  | 1.26               | 1 (August 22     | 1 (~August 22         | KubeCon + CloudNativeCon NA likely to occur |
+| 49                  | 1.26               | 14 (December 06) | 14 (~December 06)     |                                             |
+
+Initial release candidates are cut **every 4 weeks.**
+With a 15 week cadence, there will be 3 release candidates:
+- v*-rc.0
+- v*-rc.1
+- v*-rc.2
+
+After a new Kubernetes version is released, if no blocker issues are discovered
+in KubeVirt's release candidate, then it's **promoted to a full release after 5 business days.**
 
 If blockers are detected, a new release candidate is generated and will be
 promoted after giving the impacted parties enough time to validate the blocker is
 addressed.
 
-**Timeline Example: No blockers detected, release is cut from the first rc**
+**Timeline Example: No blockers detected, release is cut from the third rc**
 ```
-July 1st  - release-v0.31 branch and v0.31.0-rc.0 tag are cut
-July 8th  - promotion of v0.31.0-rc.0 to be the official v0.31.0 release
+July 19th   - release-v0.31 branch and v0.31.0-rc.2 tag are cut
```

Comment:
if the release branch is cut with rc.2, then what branch are rc.0 and rc.1 cut from? Are we saying that the release branch isn't cut until the final rc?

Reply:
Yes.  I'm thinking we use tags until we near the end of the release cycle.